packages/client/src/hooks/useChessboard.ts

```diff
@@ -84,6 +85,15 @@ const useChessboard = () => {
     return srcObj.perUrl + heroIdString + ".png";
   };
 
+  const creatureMap = new Map(
+    useEntityQuery([Has(Creature)])
+      ?.map((row) => ({
+        ...getComponentValueStrict(Creature, row),
+        key: row,
+      }))
+      .map((c) => [Number(c.key), c])
+  );
```

Comment:
In fact, the map as a cache is not nessarary at all. Use `getComponentValue` when you need value

Reply:
always have the latest data